### PR TITLE
update telegram urls

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
#### Description of the pull request (what is new / what has changed)
update telegram URLs

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

**Review by schimih — DISMISSED** [review]

File: docs/integrators/deep-history-squad.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ _Downloading_ the necessary archives and _unarchiving_ them is encapsulated in a
 
 First, choose an empty folder to serve as the workspace (working directory) of the squad instance - for example, `~/deep-history-workspace`.
 
-Afterward, prepare a configuration file called `reconstruction.json`, following the example of [`default.reconstruction.json`](https://github.com/ElrondNetwork/deep-history/tree/main), and save it in the chosen workspace. For the fields `oldestArchive` and `newestArchive`, use URLs towards the MultiversX public archive (which are available [on request](https://t.me/ElrondDevelopers)). The URLs in the example below are mere placeholders.
+Afterward, prepare a configuration file called `reconstruction.json`, following the example of [`default.reconstruction.json`](https://github.com/ElrondNetwork/deep-history/tree/main), and save it in the chosen workspace. For the fields `oldestArchive` and `newestArchive`, use URLs towards the MultiversX public archive (which are available [on request](https://t.me/MultiversXDevelopers)). The URLs in the example below are mere placeholders.
```
Comment (miiu96) [inline docs/integrators/deep-history-squad.md:62]: can you remove ElrondNetwork from this sentence?

**Review by miiu96 — APPROVED** [review]

File: docs/developers/sc-calls-format.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ erdpy wallet bech32 --encode 0139472eff6886771a982f3083da5d421f24c29181e63888228
 
 will output `erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th`.
 
-The encoding algorithm that handles these conversions can be found [here](https://github.com/ElrondNetwork/elrond-sdk-erdpy/blob/main/erdpy/wallet/bech32.py).
+The encoding algorithm that handles these conversions can be found [here](https://github.com/multiversx/mx-sdk-py-cli/blob/main/erdpy/wallet/bech32.py).
 
 ### Converting addresses using erdjs
```
Comment (schimih) [inline docs/developers/sc-calls-format.md:130]: no erdjs anymore. :disappointed:

File: docs/developers/sc-calls-format.md [hunk 3]
```diff
@@ -153,7 +153,7 @@ console.log(address.bech32());
 
 will output `erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th`.
 
-The encoding algorithm that handles these conversions can be found [here](https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/address.ts).
+The encoding algorithm that handles these conversions can be found [here](https://github.com/multiversx/mx-sdk-js-core/blob/main/src/address.ts).
 
 ### Converting addresses using erdgo
```
Comment (schimih) [inline docs/developers/sc-calls-format.md:158]: nor erdgo

File: docs/developers/sc-calls-format.md [hunk 4]
```diff
@@ -197,7 +197,7 @@ fmt.Println(addressObj.AddressAsBech32String())
 
 will output `erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th`.
 
-The encoding algorithm that handles these conversions can be found [here](https://github.com/ElrondNetwork/elrond-go-core/blob/main/core/pubkeyConverter/bech32PubkeyConverter.go).
+The encoding algorithm that handles these conversions can be found [here](https://github.com/multiversx/mx-chain-core-go/blob/main/core/pubkeyConverter/bech32PubkeyConverter.go).
 
 ### Converting addresses using erdjava
```
Comment (schimih) [inline docs/developers/sc-calls-format.md:202]: nor erdjava

File: docs/sdk-and-tools/dapp-core.md [hunk 5]
```diff
@@ -17,15 +17,15 @@ It is built for applications that use React.
...
 ### npmjs
 
 dapp-core can be found on npmjs as well: https://www.npmjs.com/package/@elrondnetwork/dapp-core
 
 ### Live demo: dapp-template
 
-The [dapp-template](https://github.com/ElrondNetwork/dapp-template/) that is used in [Build a dApp in 15 minutes](/developers/tutorials/your-first-dapp) is based on `dapp-core`.
+The [mx-template-dapp](https://github.com/multiversx/mx-template-dapp) that is used in [Build a dApp in 15 minutes](/developers/tutorials/your-first-dapp) is based on `dapp-core`.
 
 A live demo of the dapp-template is available at [https://dapp-template.elrond.com/](https://dapp-template.elrond.com/).
```
Comment (schimih) [inline docs/sdk-and-tools/dapp-core.md:30]: do we still have this one?

**Review by miiu96 — DISMISSED** [review]

**Review by schimih — APPROVED** [review]
Green for the scope of the PR.
left some comments dough, only what popped when reviewing the follow-on commits.

File: docs/developers/sc-calls-format.md (hunk 4 above)
Comment (bogdan-rosianu) [reply docs/developers/sc-calls-format.md:202]: will do in another pr

File: docs/sdk-and-tools/dapp-core.md (hunk 5 above)
Comment (bogdan-rosianu) [reply docs/sdk-and-tools/dapp-core.md]: removed

File: docs/developers/sc-calls-format.md (hunk 3 above)
Comment (bogdan-rosianu) [reply docs/developers/sc-calls-format.md:158]: will do in another pr

**Review by schimih — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**